src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,27 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int MIN_AGE = 35;
+    private static final String UKRAINIAN_NATIONALITY = "Ukrainian";
+    private static final int REQUIRED_PERIOD_OF_LIVING_IN_UKRAINE = 10;
+    private static final String SEPARATOR_FOR_PERIOD = "-";
+    private static final int START_OF_PERIOD = 0;
+    private static final int END_OF_PERIOD = 1;
```

Comment:
You can shorter your constants names but don't lose the logical context.
```suggestion
    private static final String PERIOD_SEPARATOR = "-";
    private static final int START_PERIOD = 0;
    private static final int END_PERIOD = 1;
```